book/Cargo.toml

```diff
@@ -24,3 +24,9 @@ serde = { version = "1.0", features = ["derive"] }
 tarpc = { version = "0.22", features = ["full"] }
 tokio = { version = "0.2", features = ["full"] }
 tokio-serde = { version = "0.6", features = ["json"] }
+
+[dev-dependencies]
+uuid = { version = "0.8", features = ["v4"] }
+
+[features]
+test-data = []
```

Comment:
The PR description says that the feature is named `testing`.

Reply:
Will update the description